Rubberduck.VBEEditor/Extensions/VbeExtensions.cs

```diff
@@ -62,11 +62,11 @@ public static IHostApplication HostApplication(this VBE vbe)
                 switch (reference.Name)
                 {
                     case "Excel":
-                        return new ExcelApp();
+                        return new ExcelApp(vbe);
                     case "Access":
                         return new AccessApp();
```

Comment:
It doesn't break anything. I've overloaded the HostApplication constructor, so ExcelApp() is valid, and so is ExcelApp(vbe). Unit testing still works under Excel and Word.

But, because #811 , ExcelApp() doesn't necessarily get you the *correct* instance of Excel, whereas, the overload Excel(app), does get you the right instance (and if it can't get the instance from vbe, then it gets an instance in the same way as ExcelApp().

All of the host applications could benefit from using the overloaded ctor, but I didn't want to change all of them at once, and Access, for example, won't always have a document-type VBComponent (Access forrm or report), so it doesn't get as much benefit as overloading Excel and Word ctors.